agent/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -l
+
+set -e
+
+echo "Running Atlas Agent"
+/atlas-agent --token "$CLOUD_TOKEN" &
+sleep "$WAIT_TIME"
```

Comment:
This means that if an agent gets a job on second 29 it only has a second left to esecute the job before being killed forcefully.

We need something more graceful.